jdhzzz-CT-USA.md

```diff
@@ -0,0 +1,11 @@
+# Central CT
+
+## Places to Eat
+- El Sombrero
+- Somewhere in Bangkok
+- Sax Pizza
+- Saints
+- Anthony Jacks Wood Fired Grill
```

Comment:
@jdhzzz any specific recommendations on what kind of 🍕 to get from Anthony Jacks?